chart/kubeapps/values.schema.json

```diff
@@ -1,125 +1,2886 @@
 {
-  "$schema": "http://json-schema.org/schema#",
```

Comment:
Thanks for using the tool to also generate the underlying JSON schema. That said, currently we don't want to use this schema. Just for you to know, in the latest [v2.6.0 release](https://github.com/vmware-tanzu/kubeapps/releases/tag/v2.6.0), we migrated to full json schema based form. Along with this chance, the [Bitnami Application Catalog](https://github.com/bitnami/charts/) is gonna autogenerate the schemas using this tool soon. Hence, we want to switch the schema once they also perform the change. Hope it makes sense for you.

In short, please revert the changes in this file.